src/test/java/trackr/logic/parser/DeleteTaskCommandParserTest.java

```diff
@@ -0,0 +1,36 @@
+package trackr.logic.parser;
+
+import static trackr.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static trackr.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static trackr.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static trackr.testutil.TypicalIndexes.INDEX_FIRST_OBJECT;
+
+import org.junit.jupiter.api.Test;
+
+import trackr.logic.commands.DeleteTaskCommand;
+
+/**
+ * As we are only doing white-box testing, our test cases do not cover path variations
+ * outside of the DeleteTaskCommand code. For example, inputs "1" and "1 abc" take the
+ * same path through the DeleteTaskCommand, and therefore we test only one of them.
+ * The path variation for those two cases occur inside the ParserUtil, and
+ * therefore should be covered by the ParserUtilTest.
+ */
+public class DeleteTaskCommandParserTest {
+
+    private DeleteTaskCommandParser parser = new DeleteTaskCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsDeleteCommand() {
```

Comment:
Wouldn't parse_validArgs_returnsDelete**Task**Command be a better and clearer name as result would be a DeleteTaskCommand instead of DeleteCommand object?